e2e/tests/caseManagementOrder_test.js

```diff
@@ -3,25 +3,69 @@ const standardDirectionOrder = require('../fixtures/standardDirectionOrder.json'
 const dateFormat = require('dateformat');
 
 let caseId;
+let today;
 
 Feature('Case Management Order Journey');
 
 BeforeSuite(async (I) => {
   caseId = await I.submitNewCaseWithData(standardDirectionOrder);
+  today = new Date();
 });
 
 Scenario('local authority sends agreed CMO to judge', async (I, caseViewPage, uploadCaseManagementOrderEventPage) => {
   await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+  await localAuthoritySendsAgreedCmo(I, caseViewPage, uploadCaseManagementOrderEventPage);
+  I.seeEventSubmissionConfirmation(config.applicationActions.uploadCMO);
+  caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
+  assertDraftCaseManagementOrder(I);
+});
+
+Scenario('Judge sends agreed CMO back to the local authority', async (I, caseViewPage, reviewAgreedCaseManagementOrderEventPage) => {
+  await I.navigateToCaseDetailsAs(config.judicaryUser, caseId);
+  await caseViewPage.goToNewActions(config.applicationActions.reviewAgreedCmo);
+  I.see('mockFile.pdf');
+  reviewAgreedCaseManagementOrderEventPage.selectReturnCmoForChanges();
+  reviewAgreedCaseManagementOrderEventPage.enterChangesRequested();
+  await I.completeEvent('Save and continue', {summary: 'Summary', description: 'Description'});
+  I.seeEventSubmissionConfirmation(config.applicationActions.reviewAgreedCmo);
+  caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
+  assertDraftCaseManagementOrder(I, 'Returned', 'PBA number is incorrect');
+  await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+  caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
+  assertDraftCaseManagementOrder(I, 'Returned', 'PBA number is incorrect');
+});
+
+Scenario('Judge seals and sends the agreed CMO to parties', async (I, caseViewPage, uploadCaseManagementOrderEventPage, reviewAgreedCaseManagementOrderEventPage) => {
+  await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+  await localAuthoritySendsAgreedCmo(I, caseViewPage, uploadCaseManagementOrderEventPage);
+  await I.navigateToCaseDetailsAs(config.judicaryUser, caseId);
+  await caseViewPage.goToNewActions(config.applicationActions.reviewAgreedCmo);
+  reviewAgreedCaseManagementOrderEventPage.selectSealCmo();
+  await I.completeEvent('Save and continue', {summary: 'Summary', description: 'Description'});
+  I.seeEventSubmissionConfirmation(config.applicationActions.reviewAgreedCmo);
+  caseViewPage.selectTab(caseViewPage.tabs.orders);
+  I.seeInTab(['Sealed Case Management Orders 1', 'Order'], 'mockFile.pdf');
```

Comment:
maybe for consistency with other test method should be introduced,like assertDraftCaseManagementOrder